server/settings.py

```diff
@@ -51,6 +51,7 @@
 # Application definition
 
 INSTALLED_APPS = [
+    'dockerflow.django',
```

Comment:
Could you make this optional, configured through an environment variable above? I don't think we want dockerflow for the heroku environment we're using.